cluster/ci/origin/pipelines/build/Jenkinsfile

```diff
@@ -32,6 +33,8 @@ pipeline {
         script {
           this._buildName = buildName(this)
           this._pushSecret = dockerCfgSecret(this, "builder")
```

Comment:
@csrwng won't the build get the push secret from the `builder` serviceaccount automatically? why do we need to be explicit on this?

Reply:
because we only populated secrets when builds were instantiated from buildconfigs
we fixed it recently, but I wasn't sure if it made it in to rc.0

Reply:
Could we check on that?